src/Controllers/SchemaController.js

```diff
@@ -831,7 +833,11 @@ export default class SchemaController {
         const existingFields = schema.fields;
         Object.keys(submittedFields).forEach(name => {
           const field = submittedFields[name];
-          if (existingFields[name] && field.__op !== 'Delete') {
+          if (
+            existingFields[name] &&
+            existingFields[name].type !== field.type &&
```

Comment:
`existingFields[name]?.type`

Reply:
Not here since it could produce unwanted result of deleting the field if field do not exists, here we need the 2 step since we have a `&&` combined to a `!==`